plugins/com.google.cloud.tools.eclipse.appengine.newproject/plugin.properties

```diff
@@ -7,5 +7,9 @@ gcp.category = Google Cloud Platform
 standard.wizard.name = Google App Engine Standard Java Project
 standard.wizard.description = Creates an App Engine Standard Java project.
 
-flex.wizard.name=Google App Engine Flexible Java Project
-flex.wizard.description=Creates an App Engine Flexible Java project.
+flex.war.wizard.name=Google App Engine Flexible Java WAR Project
+flex.war.wizard.description=Creates an App Engine Flexible Java project for the Jetty or Tomcat \
```

Comment:
see email I sent you off-Github re Tomcat.

Reply:
My 2¢: repeating "App Engine Flexible Java project" in the WAR and JAR descriptions doesn't help.  This WAR description should mention "servlet-based", and the JAR description should mention "runnable jar".